src/config/vim.zig

```diff
@@ -31,12 +40,18 @@ pub const ftplugin =
     \\
 ;
 pub const compiler =
+    \\" Vim compiler file
+    \\" Language: Ghostty config file
+    \\" Maintainer: Ghostty <https://github.com/ghostty-org/ghostty>
+    \\"
+    \\" THIS FILE IS AUTO-GENERATED
+    \\
     \\if exists("current_compiler")
     \\  finish
     \\endif
     \\let current_compiler = "ghostty"
     \\
-    \\CompilerSet makeprg=ghostty\ +validate-config
+    \\CompilerSet makeprg=ghostty\ +validate-config\ --config-file='%:p'
```

Comment:
`%:S` escapes special characters and is common in compiler files.

```suggestion
    \\CompilerSet makeprg=ghostty\ +validate-config\ --config-file=%:S
```